pkg/cmd/roachtest/tests/restore.go

```diff
@@ -471,7 +471,6 @@ func registerRestore(r registry.Registry) {
 		{dataSet: dataBank2TB{}, nodes: 10, timeout: 6 * time.Hour},
 		{dataSet: dataBank2TB{}, nodes: 32, timeout: 3 * time.Hour},
 		{dataSet: dataBank2TB{}, nodes: 6, timeout: 4 * time.Hour, cpus: 8, largeVolumes: true},
-		{dataSet: tpccIncData{}, nodes: 10, timeout: 6 * time.Hour},
```

Comment:
if we're not running tpccIncData do you think you could:

1. Remove tpccIncData
2. Rename tpccIncDataLatest back to tpccIncData

I had only added latest cause we didn't want to delete tpccIncData as there were some ongoing invesitgations.

Reply:
I think this would also mean that it'd reflect correctly in roachperf.

Reply:
aha. Good point. done.